### ISSUE TITLE
It doesn't work

### ISSUE THREAD

**Issue by yhsiah** [akshaybajaj2280/wtfsigte#2]
maps integration seems busted

**Comment by kendoman96** [comment]
I don't believe the site is maintained anymore, sadly. It will throw an error along the lines of "the page cannot load google maps correctly"

also ssl cert expired the first of this month
